index.bs

```diff
@@ -4681,12 +4695,11 @@ In order to perform a [=registration ceremony=], the [=[RP]=] MUST proceed as fo
 1. If [=user verification=] is required for this registration, verify that the [=User Verified=] bit of the <code>[=flags=]</code>
     in |authData| is set.
 
-1. If the credential [=backup eligibility=] is used as part of Relying Party business logic or policy, evaluate the 
+1. If the [=[RP]=] uses the credential's [=backup eligibility=] to inform its user experience flows and/or policies, evaluate the 
     [=backup eligibility=] bit of the <code>[=flags=]</code> in |authData|.
```

Comment:
Adding the abbreviated name here makes the connection to the flags table clearer.

```suggestion
    [=backup eligibility=] (BE) bit of the <code>[=flags=]</code> in |authData|.
```